packages/@react-spectrum/contextualhelp/src/ContextualHelp.tsx

```diff
@@ -40,23 +41,19 @@ function ContextualHelp(props: SpectrumContextualHelpProps, ref: FocusableRef<HT
     footer: {UNSAFE_className: helpStyles['react-spectrum-ContextualHelp-footer']}
   };
 
-  let ariaLabel = otherProps['aria-label'];
-  if (!ariaLabel && !otherProps['aria-labelledby']) {
-    ariaLabel = stringFormatter.format(variant);
-  }
+  let labelProps = useLabels(otherProps, stringFormatter.format(variant));
 
   return (
     <DialogTrigger {...otherProps} type="popover" placement={placement} hideArrow>
       <ActionButton
-        {...otherProps}
+        {...mergeProps(otherProps, labelProps)}
         ref={ref}
-        UNSAFE_className={helpStyles['react-spectrum-ContextualHelp-button']}
-        isQuiet
-        aria-label={ariaLabel}>
+        UNSAFE_className={classNames(helpStyles, 'react-spectrum-ContextualHelp-button', otherProps.UNSAFE_className)}
```

Comment:
Why not `styleProps.className` from `useStyleProps` here?

Reply:
ActionButton will call useStyleProps for us, no reason to call it at this level